api/src/pcapi/core/educational/api/redactor.py

```diff
@@ -0,0 +1,9 @@
+from typing import Any
+
+from pcapi.core.educational import models
+from pcapi.repository import repository
+
+
+def save_redactor_preferences(redactor: models.EducationalRedactor, **preferences: Any) -> None:
+    redactor.preferences = {**redactor.preferences, **preferences}
```

Comment:
(style / question de goût)

```suggestion
    redactor.preferences |= preferences
```

Reply:
Pour une raison que je ne comprends toujours pas, avec cette option (ou un `.update()`), les préférences ne sont pas enregistrées. Est-ce parce que SQLA ne détecte pas une modification l'objet `redactor` et donc qu'il lance un INSERT avec la valeur par défaut ?...